proposals/acp-specification/acp.ttl

```diff
@@ -0,0 +1,244 @@
+prefix acp: <http://www.w3.org/ns/solid/acp#>
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix dc: <http://purl.org/dc/terms/>
+prefix vann: <http://purl.org/vocab/vann/>
+
+<http://www.w3.org/ns/solid/acp#>
```

Comment:
It is and I think it should always be the way by default, especially because the above namespace doesn't support 303 redirects. So <http://www.w3.org/ns/solid/acp> is a dereferenceable HTTP resource/document and <http://www.w3.org/ns/solid/acp#> is an Ontology.

For what it's worth, it is the way [RDF](http://www.w3.org/1999/02/22-rdf-syntax-ns#), [RDFS](http://www.w3.org/2000/01/rdf-schema#) and [OWL](http://www.w3.org/2002/07/owl#) do it.

Reply:
OWL uses the document without fragment but indeed RDF and RDFS both use empty fragment. Resolving...